app/src/main/java/com/android/unio/ui/authentication/AccountDetails.kt

```diff
@@ -159,33 +180,60 @@ fun AccountDetails(
         }
         OutlinedButton(
             modifier = Modifier.fillMaxWidth().testTag("AccountDetailsSocialsButton"),
-            onClick = {
-              Toast.makeText(context, "Not yet implemented", Toast.LENGTH_SHORT).show()
-            }) {
+            onClick = { showSocialsOverlay = true }) {
               Icon(Icons.Default.Add, contentDescription = "Add")
               Text("Add links to other social media")
             }
-        FlowRow() {
-          /* TODO row containing dynamic list of social media links */
+        FlowRow(modifier = Modifier.fillMaxWidth()) {
+          socials.forEachIndexed { index, userSocial ->
+            InputChip(
+                label = { Text(userSocial.social.name) },
+                onClick = {},
+                selected = true,
+                modifier = Modifier.padding(3.dp).testTag("AccountDetailsSocialChip: $index"),
+                avatar = {
+                  Icon(
+                      Icons.Default.Close,
+                      contentDescription = "Add",
+                      modifier =
+                          Modifier.clickable {
+                            userSocialsFlow.value =
+                                userSocialsFlow.value.toMutableList().apply { removeAt(index) }
+                          })
+                })
+          }
         }
         Button(
             modifier = Modifier.testTag("AccountDetailsContinueButton"),
             onClick = {
-              val user =
-                  User(
-                      uid = Firebase.auth.currentUser?.uid!!,
-                      email = Firebase.auth.currentUser?.email!!,
-                      firstName = firstName,
-                      lastName = lastName,
-                      biography = bio,
-                      followedAssociations = Association.emptyFirestoreReferenceList(),
-                      joinedAssociations = Association.emptyFirestoreReferenceList(),
-                      interests = interests.filter { it.second.value }.map { it.first },
-                      socials = emptyList(),
-                      profilePicture = "",
-                      hasProvidedAccountDetails = true)
-              uploadUser(user, userRepositoryFirestore, navigationAction, context)
-              navigationAction.navigateTo(Screen.HOME)
+              if (firstName.isEmpty() ||
```

Comment:
Minor fix : Consider reloading error messages when clicking again on the create button, as I had an issue when running this part :  
I willingly did not put a name and a surname so they became red and ask me to fill them both before continuing. I did fullfilled the name and then click again on the create button. However, both boxes were still in red.

It can lead to confusion for the user. When they correct their input and attempt to proceed, they expect the UI to reflect their changes. If the fields remain highlighted in red, it suggests that the input is still invalid, causing frustration and potentially leading users to believe there is a bug or that their input is still incorrect. Implementing a refresh of the error states upon each attempt to submit would enhance user experience by providing immediate feedback and clarity on the input status.